aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -854,14 +875,22 @@ mod tests {
 
         let mut base_update = BTreeMap::new();
         let mut additional_update = BTreeMap::new();
-        base_update.insert(key.clone(), GroupWrite {
-            metadata_op: write_op_with_metadata(0, 100), // create
-            inner_ops: BTreeMap::new(),
-        });
-        additional_update.insert(key.clone(), GroupWrite {
-            metadata_op: write_op_with_metadata(2, 200), // delete
-            inner_ops: BTreeMap::new(),
-        });
+        base_update.insert(
+            key.clone(),
+            GroupWrite::new(
+                write_op_with_metadata(0, 100), // create
+                BTreeMap::new(),
+                0,
+            ),
+        );
+        additional_update.insert(
+            key.clone(),
+            GroupWrite::new(
+                write_op_with_metadata(2, 200), // delete
```

Comment:
nit: not sure if this is worth doing for tests or not but you can also do `const DELETE: u64 = 2`  to make test cases slightly more readable